pc

```diff
@@ -143,6 +143,7 @@ current_branch=${current_branch##refs/heads/}
 
 function update_deploy_branches {
   git fetch
+  git fetch --tags
```

Comment:
D'après la doc : "'Fetch all tags [...] in addition to whatever else would otherwise be fetched."
Donc j'ai envie de dire que tu peux enlever la ligne 145 !

Reply:
et ben je l'enlève vu que ça fait les deux :)